examples/QIR/Development/Development.csproj

```diff
@@ -8,18 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Condition="$([MSBuild]::IsOsPlatform('Windows'))"
-                      Include="libLLVM.runtime.win-x64" Version="11.0.0"
-                      PrivateAssets="All" GeneratePathProperty="true" />
-    <PackageReference Condition="$([MSBuild]::IsOsPlatform('OSX'))"
-                      Include="libLLVM.runtime.osx-x64" Version="11.0.0"
-                      PrivateAssets="All" GeneratePathProperty="true" />
-    <PackageReference Condition="$([MSBuild]::IsOsPlatform('Linux')) And '$(UbuntuVersion)' == '18.04'"
-                      Include="libLLVM.runtime.ubuntu.18.04-x64" Version="11.0.0"
-                      PrivateAssets="All" GeneratePathProperty="true" />
-    <PackageReference Condition="$([MSBuild]::IsOsPlatform('Linux')) And '$(UbuntuVersion)' != '18.04'"
-                      Include="libLLVM.runtime.ubuntu.20.04-x64" Version="11.0.0"
-                      PrivateAssets="All" GeneratePathProperty="true" />
+    <PackageReference Include="Microsoft.Quantum.LlvmBindings.Native" Version="13.0.0-CI-20220129-005156" PrivateAssets="All" GeneratePathProperty="true"/>
```

Comment:
Just curious, why does the `Microsoft.Quantum.LlvmBindings.Native` package has a `Native` suffix?